architecture/additional_concepts/storage.adoc

```diff
@@ -192,11 +192,23 @@ For example, NFS can support multiple read/write clients, but a specific NFS PV
 might be exported on the server as read-only. Each PV gets its own set of access
 modes describing that specific PV's capabilities.
 
-Claims are matched to Volumes with similar access modes. A claim's access modes
-represent a request. Therefore, the user may be granted more, but never less.
-For example, if a claim requests RWO, but the only volume available was an NFS PV
-(RWO+ROX+RWX), the claim would match NFS because it supports RWO. Direct matches
-are always attempted first.
+Claims are matched to Volumes with similar access modes. The only two matching
+criteria are access modes and size. A claim's access modes represent a request.
+Therefore, the user may be granted more, but never less. For example, if a claim
+requests RWO, but the only volume available was an NFS PV (RWO+ROX+RWX), the
+claim would match NFS because it supports RWO.
+
+Direct matches are always attempted first. The modes must match or contain more
```

Comment:
Just for clarity, call out who owns the modes we're looking at?

  "the volume's modes must match or contain..."